src/app/agent/entities/Agent.ts

```diff
@@ -0,0 +1,8 @@
+import { Booking } from './Booking';
+
```

Comment:
Consider putting these in a common place where we can all use them so we do not duplicate code.